etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,38 @@
+
+# Firejail profile for Parsec
+# Description: Remote desktop application focused on gaming and other 3D applications
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include parsecd.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.parsec
+whitelist ${HOME}/.parsec
+whitelist /usr/share/parsec
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
```

Comment:
What about more `whitelist-*-common.inc`?

Reply:
Why? It has no filesharing capability and does not need access to more paths

Reply:
Greate, then let's restict it

```
include whitelist-run-common.inc
include whitelist-runuser-common.inc
include whitelist-var-common.inc
```

Reply:
Hm, since it doesnt need those paths, can we just blacklist /var entirely?

Reply:
Correction, it doesn't need those paths *on your system*. Some of those paths are dependent on distribution/configuration/hardware/...

Reply:
This is correct but I dont see parsec looking for those paths in the strace for example